Add First Steps topic (#7512)
DevExpress/devextreme-documentation: concepts/Common/First Steps/20 Show Progress/00 Show Progress.md

> By this point of the tutorial, you built a task DataGrid that allows record editing. To display task progress, you integrated a ProgressBar in the layout. The next steps are to configure the ProgressBar options and synchronize it with DataGrid updates.

Reviewer: Are "users" allowed to edit records? If so, can we update this to:

```suggestion
At this point of the tutorial, you have built a task DataGrid that allows users to edit records. To display task progress, you integrated a ProgressBar into the layout. The next steps are to configure the ProgressBar options and synchronize it with DataGrid updates.
```
